src/app/featureFlags/selectors.ts

```diff
@@ -0,0 +1,24 @@
+import { createSelector } from 'reselect';
+import { book, bookTheme } from '../content/selectors';
+import { BookWithOSWebData } from '../content/types';
+import * as parentSelectors from '../selectors';
+
+export const enabled = createSelector(
+  parentSelectors.localState,
+  (parentState) => parentState.featureFlags
+);
+
+export const searchButtonStyle = createSelector(
+  enabled,
+  (featureFlags) => featureFlags.searchButton || null
+);
+
+export const searchButtonColor = createSelector(
```

Comment:
@PiotrKozlowski is this what you had in mind?

Reply:
yes!